_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,277 @@
+| `@type`             | "ShareAttributeRequestItem"                                                                                                                                                  | Specifies the type of the Request item for parsing.                                                                                                                                                                             |
+| `attribute`         | [IdentityAttribute]({% link _docs_explore/61-data-model.md %}#identityattribute) \| [RelationshipAttribute]({% link _docs_explore/61-data-model.md %}#relationshipattribute) | The IdentityAttribute or RelationshipAttribute to share. This is not the LocalAttribute but the content data structure of the attribute. <br>The owner of the Attribute which should be shared can only be the sender Identity. |
+| `sourceAttributeId` | `string`                                                                                                                                                                     | The id of the LocalAttribute which is the source of the shared Attribute. This will be used later to reference the sourceAttribute from its shared copy.                                                                        |
+
+### Response
+
+#### Item Properties
+
+| Name          | Type                               | Description                      |
+| ------------- | ---------------------------------- | -------------------------------- |
+| `@type`       | "ShareAttributeAcceptResponseItem" | The type of the ResponseItem.    |
+| `attributeId` | `string`                           | The id of the created Attribute. |
+
+### Combinations and usage scenarios
+
+| Attribute Type | Attribute Owner | Possible? | Automation    | Examples/Reason                                                       |
+| -------------- | --------------- | --------- | ------------- | --------------------------------------------------------------------- |
+| Identity       | Sender          | Y         | `AUTO ACCEPT` | Company sends new customer the address of the company                 |
+| Identity       | Recipient       | N         |               | You cannot share an Attribute of which you are not the owner.         |
+| Identity       | `<empty>`       | Y         | `AUTO ACCEPT` | An empty owner defaults to an Attribute with owner=`<sender>`.        |
+| Relationship   | Sender          | Y         |               | Share Customer ID from Company A with Company B (e.g. Payback number) |
+| Relationship   | Recipient       | N         |               | You cannot share an Attribute of which you are not the owner.         |
+| Relationship   | `<empty>`       | Y         | `AUTO ACCEPT` | An empty owner defaults to an Attribute with owner=`<sender>`.        |
```

Comment:
In der Tabelle fehlen einige Werte in der "Automation" Spalte. Am besten nochmal im ganzen Dokument checken.